src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from "react";
+import '../../../src/App.scss';
+
+export const Person = ({ person }) => {
+    return (
+        <section className="Person">
+            <h2 className="Person__name">My name is {person.name}</h2>
+            {person.age 
+            ? <p className="Person__age">I am {person.age}</p> 
+            : ''}
+            {person.isMarried 
+            ? <p className="Person__partner">{person.partnerName} is my {person.sex === 'm'
+            ? 'wife' 
+            : 'husband'}</p> 
+            : 'I am not married'}
```

Comment:
Make sure to pass all tests 
<img width="963" alt="Screenshot 2024-09-03 at 14 31 35" src="https://github.com/user-attachments/assets/0d37be3f-5527-49af-b14b-aada6c8c4c7e">